### PR TITLE
Ensure upgrades of curl dependencies

### PR DESCRIPTION
**Pull request by aharden**
Solution to errors seen in actions which include curl:
`Error relocating /usr/bin/curl: curl_multi_poll: symbol not found
##[error]Docker run failed with exit code 127`

### CODE REVIEW

File: apply/Dockerfile [hunk 1]
```diff
@@ -9,7 +9,8 @@ LABEL "repository"="https://github.com/hashicorp/terraform-github-actions"
 LABEL "homepage"="http://github.com/hashicorp/terraform-github-actions"
 LABEL "maintainer"="HashiCorp Terraform Team <terraform@hashicorp.com>"
 
-RUN apk --no-cache add jq curl
+RUN apk add --update-cache --upgrade curl
+RUN apk --no-cache add jq
```
Comment (sodabrew) [inline apply/Dockerfile]: I would suggest to keep `jq` and `curl` on the same command-line still. Why split them?

Comment (aharden) [reply apply/Dockerfile]: I split them just to make sure the change was localized to fix curl.

Comment (sudomateo) [reply apply/Dockerfile]: I'm leaning towards this.
```
RUN apk --upgrade --no-cache add jq curl 
```

Comment (aharden) [reply apply/Dockerfile]: OK.  For context: I grabbed the recommended curl fix from here: https://github.com/curl/curl/issues/4357#issuecomment-532139456

I'll try out `RUN apk --upgrade --no-cache add jq curl`.

Comment (aharden) [reply apply/Dockerfile]: @sudomateo That looks good.  I'll close this PR and open a fresh one with that change.

**Pull request closed without merging**